tests/unit/presenters/slack/enhanced-data-presenter-test.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Presenters\Slack;
+
+use Brain\Monkey\Functions;
+use Yoast\WP\SEO\Presentations\Indexable_Presentation;
+use Yoast\WP\SEO\Presenters\Slack\Enhanced_Data_Presenter;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+/**
+ * Class Enhanced_Data_Presenter_Test.
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Presenters\Slack\Enhanced_Data_Presenter
+ *
+ * @group presenters
+ * @group slack
+ */
+class Enhanced_Data_Presenter_Test extends TestCase {
```

Comment:
Missing test for the `get` method.

Reply:
It was meant to be covered by `test_present()`:
https://github.com/Yoast/wordpress-seo/blob/16b01393ec28870b35eb207c11166edb42ae0946/tests/unit/presenters/slack/enhanced-data-presenter-test.php#L50

Most of the presenters groups test for `present()` with `get()`, it seemed reasonable to me to do the same here since `get()` doesn't really add any logic (it's just assigning values from a WP function and `get_reading_time()`) and, most important, _a separate test for `get()` would be a duplicate of the test that's already there_ (since `present()` calls `get()` which we can't mock).
I'll add a tag in the PHPDoc to clarify that it covers `get_reading_time()` too, anyway.